events/2025-09-17-ws-historikertag.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: event-external
+title: Galaxy: The Open Source Platform as an entry point to Digital Humanities and Research Data Management
```

Comment:
```suggestion
title: "Galaxy: The Open Source Platform as an entry point to Digital Humanities and Research Data Management"
```

since the colon `:` has special meaning in yaml, you should put it in quotes if it is part of your title